### ISSUE TITLE
Insufficient Contrast between Help Text and Background on Document Upload Page

### ISSUE THREAD

**Issue by Meng87** [CMU-313/Mayan-EDMS#90]
When uploading documents, the help text and drop down menu text have insufficient contrast with the background. Currently, the accessibility score is 88.
